### PR TITLE
feat(edge): add FAQ

### PR DESCRIPTION
**Pull request by RoRoJ**
Add FAQ for Edge Services.

The TODO for the hero will be fixed before going into prod (currently merging into feature branch)

### CODE REVIEW

File: faq/edge-services.mdx [hunk 1]
```diff
@@ -10,4 +10,20 @@ dates:
 category: network
 ---
 
-TODO ADD HERO
\ No newline at end of file
+TODO ADD CORRECT HERO
+
+## What is Edge Services?
+
+Edge Services is an feature for Scaleway Load Balancers and Object Storage buckets, currently available in [Public Beta](https://www.scaleway.com/en/betas/). It provides a [caching service](/network/edge-services/how-to/configure-cache/) to improve performance by reducing load on your [origin](/network/edge-services/concepts/#origin), and a customizable and secure endpoint for accessing content via Edge Services, which can be set to a subdomain of your choice.
```
Comment (bene2k1) [inline faq/edge-services.mdx:17]: Can it be only a subdomain or can I use it also on my primary domain ( cache.example.com vs exaple.com ) ?

Comment (RoRoJ) [reply faq/edge-services.mdx:17]: It has to be a subdomain, can't be a primary domain. This is specified in the main doc, do you think it should be said here too?

Comment (bene2k1) [reply faq/edge-services.mdx:17]: no, it is fine then. I was just not sure if it is possible 👍

**Review by bene2k1 — APPROVED** [review]

File: faq/edge-services.mdx (hunk 1 above)
Comment (nerda-codes) [inline faq/edge-services.mdx:17]:
```suggestion
Edge Services is a feature for Scaleway Load Balancers and Object Storage buckets, currently available in [Public Beta](https://www.scaleway.com/en/betas/). It provides a [caching service](/network/edge-services/how-to/configure-cache/) to improve performance by reducing load on your [origin](/network/edge-services/concepts/#origin), and a customizable and secure endpoint for accessing content via Edge Services, which can be set to a subdomain of your choice.
```

File: faq/edge-services.mdx [hunk 2]
```diff
@@ -10,4 +10,20 @@ dates:
 category: network
 ---
 
-TODO ADD HERO
\ No newline at end of file
+TODO ADD CORRECT HERO
```
Comment (nerda-codes) [inline faq/edge-services.mdx:13]: reminder here

File: faq/edge-services.mdx [hunk 3]
```diff
@@ -4,10 +4,24 @@ meta:
...
+
+## Which products are compatible with Edge Services?
+
+Edge Services is currently available for Scaleway [Object Storage buckets](/storage/object/) and Scaleway [Load Balancers](/network/load-balancer/).
+
+## How much does Edge Services cost?
+
+Since Edge Services is currently in Public Beta, it is free to use and not billed. Users will be warned before it goes into General Availability and becomes billable. Pricing will be based on subscription plans.
+
+## If I customize by Edge Services endpoint with my own domain, can it serve content over HTTPS?
```
Comment (jcirinosclwy) [inline faq/edge-services.mdx:25]:
```suggestion
## If I customize my Edge Services endpoint with my own domain, can it serve content over HTTPS?
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**